telemetry/sql/000014_device_type.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE peercount ADD COLUMN deviceType VARCHAR(255);
+ALTER TABLE receivedMessages ADD COLUMN deviceType VARCHAR(255);
+ALTER TABLE receivedEnvelopes ADD COLUMN deviceType VARCHAR(255);
+ALTER TABLE sentEnvelopes ADD COLUMN deviceType VARCHAR(255);
+ALTER TABLE errorSendingEnvelope ADD COLUMN deviceType VARCHAR(255);
+ALTER TABLE peerConnFailure ADD COLUMN deviceType VARCHAR(255);
```

Comment:
Do we still need to add `deviceType` to all tables?

Reply:
Keeping it in as it's included in separate PR that will be merged first https://github.com/status-im/telemetry/pull/44